### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: Helper/Recommended.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Recommended extends \Magento\Framework\App\Helper\AbstractHelper
 
     const XML_PATH_RELATED_PRODUCTS_ITEMS = 'connector_dynamic_content/products/related_items_to_display';
     const XML_PATH_UPSELL_PRODUCTS_ITEMS = 'connector_dynamic_content/products/upsell_items_to_display';
-    const XML_PATH_CROSSSELL_PRODUCTS_ITEMS = 'connector_dynamic_content/products/crosssell_items_to_display';
+    const XML_PATH_CROSSSELL_PRODUCTS_ITEMS = 'connector_dynamic_content/products/crossell_items_to_display';
```
Comment (cdiacon) [inline Helper/Recommended.php]: CROSSSELL as const and crossell in value?

@adeelq

Comment (adeelq) [inline Helper/Recommended.php]: @cdiacon did not check the const name..gonna change that too cheers

Comment (cdiacon) [inline Helper/Recommended.php]: no, you have to revert, the right spelling will be : crosssell.

Comment (adeelq) [inline Helper/Recommended.php]: just did i figure it out later on 👍

**Pull request merged**